pay/advance.go

```diff
@@ -63,7 +74,7 @@ func (a *Advance) CalculateFrom(totalWithTax num.Amount) {
 // UnmarshalJSON helps migrate the desc field to description.
 func (a *Advance) UnmarshalJSON(data []byte) error {
 	type Alias Advance
-	aux := &struct {
```

Comment:
Same as before: this change should better be part of https://github.com/invopop/gobl/pull/234